Subject: [PATCH] Don't build-depend on libgcrypt, as nothing is used from it

Backported from http://www.cups.org/strfiles.php/3308/cups-no-gcrypt.patch

This addresses the cryto dependency seen during build.

Upstream-Status: Backport

Signed-off-by: Jackie Huang <jackie.huang@windriver.com>
---
 cups/libs/cups/http-private.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/cups/libs/cups/http-private.h b/cups/libs/cups/http-private.h
index 9c88281..94a9817 100644
--- a/cups/libs/cups/http-private.h
+++ b/cups/libs/cups/http-private.h
@@ -98,7 +98,6 @@ extern BIO_METHOD *_httpBIOMethods(void);
  * The GNU TLS library is more of a "bare metal" SSL/TLS library...
  */
 #    include <gnutls/gnutls.h>
-#    include <gcrypt.h>
 
 typedef struct
 {
-- 
2.0.0

